docs/source/task-launch.rst

```diff
@@ -58,21 +58,54 @@ on ``data`` remotely and then break up data into its various elements.
 
 .. code-block:: python
 
-   >>> n = e.submit(len, data)                 # compute number of elements
+   >>> n = client.submit(len, data)            # compute number of elements
    >>> n = n.result()                          # gather n (small) locally
 
    >>> from operator import getitem
-   >>> elements = [e.submit(getitem, data, i) for i in range(n)]  # split data
+   >>> elements = [client.submit(getitem, data, i) for i in range(n)]  # split data
 
-   >>> futures = e.map(process_element, elements)
-   >>> analysis = e.submit(aggregate, futures)
+   >>> futures = client.map(process_element, elements)
+   >>> analysis = client.submit(aggregate, futures)
 
 We compute the length remotely, gather back this very small result, and then
 use it to submit more tasks to break up the data and process on the cluster.
 This is more complex because we had to go back and forth a couple of times
 between the cluster and the local process, but the data moved was very small,
 and so this only added a few milliseconds to our total processing time.
 
+Extended Example
+~~~~~~~~~~~~~~~~
+
+This example computing the Fibonacci numbers creates tasks that submit tasks
+that submit tasks that submit other tasks, etc. This example is a good example
+of when the APIs below should be used.
+
+.. code-block:: python
+
+   In [1]: from distributed import Client
+
+   In [2]: client = Client()
+
+   In [3]: def fib(n):
+      ...:     if n < 2:
+      ...:         return n
+      ...:     a = fib(n - 1)  # we want to submit this function to the client
+      ...:     b = fib(n - 2)  # we want to submit this function to the client
+      ...:     return a + b
+
+   In [4]: future = client.submit(fib, 100)
```

Comment:
Does this actually finish?  `fib(100)` is pretty crazy

Reply:
Looks like it got left in there by mistake. The old docs used `worker_client` in this function.

Reply:
Right, the text just below this is also a bit odd.  It makes it sound like we're using Dask here but we aren't.  We might avoid all use of Dask in this first example of "the thing we're trying to parallelize"